sdk/openai/Azure.AI.OpenAI/api/Azure.AI.OpenAI.netstandard2.0.cs

```diff
@@ -620,7 +623,9 @@ public partial class ChatMessageImageContentItem : Azure.AI.OpenAI.ChatMessageCo
     }
     public partial class ChatMessageImageUrl : System.ClientModel.Primitives.IJsonModel<Azure.AI.OpenAI.ChatMessageImageUrl>, System.ClientModel.Primitives.IPersistableModel<Azure.AI.OpenAI.ChatMessageImageUrl>
     {
+        public ChatMessageImageUrl(string dataUri) { }
         public ChatMessageImageUrl(System.Uri url) { }
+        public string DataUri { get { throw null; } }
```

Comment:
ImageUri and it must be System.Uri, not string

Reply:
`System.Uri` will not work. please read the message on the PR why this has to be a string.